src/main/java/propertyFiles/citation.properties

```diff
@@ -22,9 +22,10 @@ datasetfieldtype.dsDescriptionValue.title=Text
 datasetfieldtype.dsDescriptionDate.title=Date
 datasetfieldtype.subject.title=Subject
 datasetfieldtype.keyword.title=Keyword
+datasetfieldtype.keywordTermURI.title=Term URI
 datasetfieldtype.keywordValue.title=Term
 datasetfieldtype.keywordVocabulary.title=Controlled Vocabulary Name
-datasetfieldtype.keywordVocabularyURI.title=Controlled Vocabulary URL
+datasetfieldtype.keywordVocabularyURL.title=Controlled Vocabulary URL
```

Comment:
Why does this need to change to URL from URI? (While you're adding Term `URI`?) Minimally I think we should be consistent and, unless we're sure that there are no URNs in use, sticking with URI might be the better option. (I do see that we focus on URLs in the descriptions and the idea of web locations for things instead of talking about URIs as identifiers that might also be resolvable on the web. I'm not sure what the best way to resolve that is, or that it needs to be handled in this PR.) @jggautier - any thoughts?

Reply:
I'm not sure and might not understand this enough. But maybe I could help see if there _are_ URNs in use?

In case this info is helpful, I think we used "keywordVocabularyURI" for the database name because [DDI Codebook uses "vocabURI"](https://ddialliance.org/Specification/DDI-Codebook/2.5/XMLSchema/field_level_documentation_files/schemas/codebook_xsd/elements/keyword.html). And maybe we thought that most people would be more familiar with the term "URLs", so that's what we used for the label people see on the page and in the tooltips. Of course this was all before I joined Dataverse.

Reply:
Update is done @qqmyers :)